dnsext-full-resolver/DNS/Cache/Log.hs

```diff
@@ -48,15 +49,21 @@ outputHandle o = case o of
   Stdout  ->  stdout
   Stderr  ->  stderr
 
-new :: Handle -> Level -> IO (IO (), Level -> [String] -> IO (), IO (Int, Int))
+new :: Handle -> Level -> IO (IO (), Level -> [String] -> IO (), IO (Int, Int), IO ())
```

Comment:
Would you define type synonyms for each IO action such as `Flush`?

Reply:
https://github.com/kazu-yamamoto/dnsext/pull/17/commits/258244b20181f1e663f80997dc586341678a56d8